config/application.rb

```diff
@@ -51,6 +51,9 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :delayed_job
     config.action_mailer.deliver_later_queue_name = :mailers
 
+    config.active_storage.variant_processor = :mini_magick
+    config.active_storage.variable_content_types = %w(image/png image/gif image/jpeg image/tiff image/bmp image/webp image/avif image/heic image/heif)
```

Comment:
this is probably the change right?

Reply:
Yes, this is the change. 
That's so weird, not sure what happened.  I'm going to close and remake the PR.